### ISSUE TITLE
Setup doesn't install rsync if missing

### ISSUE THREAD

**Issue by Ilnahro** [mail-in-a-box/mailinabox#2065]
I was pulling my hair out today when, after a migration of my server, setting up my rsync backup continue to throw up an obscure error message that I could find no reference to anywhere.

By chance I tried installing rsync (sudo apt install rsync) and to my surprise, the package was indeed missing. By the time I found this, I had already tried to fix this for several hours, including two attempts at migration.

rsync seems like a dependency that should be installed.

**Comment by myfirstnameispaul** [comment]
Wherever you installed this to does not meet the minimum requirements for Mail-in-a-Box. The Official Ubuntu Server 18.04 manifest includes `rsync`, and a fresh install of Ubuntu Server 18.04 is listed as a minimum requirement, therefore this is not issue with the Mail-in-a-Box project.

**Comment by yodax** [comment]
We'll we've made changes for specific cloud images before. I know the default Scaleway image doesn't include rsync. (or at least it didn't)

I think it would be good to add the rsync package as a dependency to the backup configuration.

**Comment by Ilnahro** [comment]
I installed this to a VPS with a fresh/updated Ubuntu 18.04 install. Clearly, this wasn't a stock install (not that any VPS providers ever provide a warning to this effect), but in my experience modifications to those images are quite commonplace in the VPS space (every provider trying to save on cost in their own way, I guess). This is the first time for me that a provider removed something critical to maib, but it would be nice if it were added as a dependency even if strictly speaking, Ubuntu 18.04 _should_ include it.

**Comment by JoshData** [comment]
Yes of course we can simply add rsync to the list of packages to install. I'd happily accept a PR that adds it at https://github.com/mail-in-a-box/mailinabox/blob/main/setup/management.sh#L28.

**Comment by myfirstnameispaul** [comment]
Indeed, I had no idea they would do this. I've certainly seen variations on configurations, but not removal of manifest packages.

**Comment by Ilnahro** [comment]
@JoshData PR is submitted
